cddl/cbor-tags.txt

```diff
@@ -16,3 +14,4 @@ tagged-cert-thumbprint-type = #6.559(digest)
 tagged-bytes = #6.560(bytes)
 tagged-cert-path-thumbprint-type = #6.561(digest)
 tagged-pkix-asn1der-cert-type = #6.562(bstr)
+tagged-integral-predicate= #6.563(integral-predicate)
```

Comment:
This looks spurious.

Reply:
Removed.

Reply:
Sure?

Reply:
Oh it was an unadded file on my side that `make` readded. Removed.